src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Update/Endpoint.cs

```diff
@@ -19,13 +19,12 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
         var existingMt = await dbContext.ResourceContentVersionMachineTranslations.AsTracking()
             .FirstOrDefaultAsync(x =>
                     x.Id == req.Id &&
-                    x.ResourceContentVersion.AssignedUserId == user.Id &&
```

Comment:
The problem with this is now anyone can update it. It's not a realistic scenario, but someone could go through and set a rating for everything in this status regardless of if they've worked on it or looked at it.

Reply:
Do we need to lock it down to specific roles/permissions?

Reply:
Assigned editor is who sets it, so everyone has the permission.

Reply:
There's some sort of timing issue where the front-end reassigns before this gets set. I think the issue is on the front-end.